build-system-tests/templates/components/react/next/App.js

```diff
@@ -0,0 +1,51 @@
+import React, { useEffect } from 'react';
+import { Amplify, Notifications } from 'aws-amplify';
+import {
+  AccountSettings,
+  Authenticator,
+  FileUploader,
+  MapView,
+  Text,
+  InAppMessagingProvider,
+  InAppMessageDisplay,
+} from '@aws-amplify/ui-react';
+import { StorageManager } from '@aws-amplify/ui-react-storage';
+import '@aws-amplify/ui-react/styles.css';
+import awsconfig from '@/data/aws-exports';
```

Comment:
Can we use relative path instead of `@/`?

Reply:
`@/` was from the auto-generated config. Since we try to mimic the users behavior, it might be better not change it

Reply:
Understood, not blocking then.

Reply:
I'm good with leaving `tsconfig` as-is then, but let's use relative path in the import statement, since that's what we use in [docs](https://ui.docs.amplify.aws/react/connected-components/authenticator#step-3-add-the-authenticator).

Reply:
That's a good catch. The newer version of Next.js auto generated the relative path as default, so I kept it here